locale/en/LC_MESSAGES/django.po

```diff
@@ -627,3 +621,59 @@ msgstr "first name"
 #: users/models.py:12
 msgid "last name"
 msgstr "last name"
+
```

Comment:
idk what this stuff is?

Reply:
Anything marked fuzzy is the translation system attempting to match a new msgid with an existing msgstr.

As long as you have another entry for "msgid "Enter a valid email address." etc it should be fine to remove all this.

https://www.gnu.org/software/gettext/manual/html_node/Fuzzy-Entries.html

Reply:
Btw I'm still not sure if we wouldn't be better off using symbolic msgids instead of mapping strings to strings. The current setup seems quite confusing.

Reply:
those fuzzy entries are only present as fuzzy, so I will leave them for now.